runtime/gc_base/RootScanner.hpp

```diff
@@ -434,7 +434,16 @@ class MM_RootScanner : public MM_BaseVirtual
 	 * which modifies elements within the list.
 	 */
 	virtual void scanOwnableSynchronizerObjects(MM_EnvironmentBase *env);
+	/**
+	 * scanContinuationObjects is for pruning continuation Objects, which are dead or last unmounted,
+	 * rom the continuation lists within a scope of current GC(for example just for Nursery in Scavenge).
+	 */
 	virtual void scanContinuationObjects(MM_EnvironmentBase *env);
+	/**
+	 * iterateAllContinuationObjects is for iterating all live Contunuation Objects in the heap and doing any processing
+	 * that other parties may register through a hook (iterateAllContinuationObjects}.
```

Comment:
did you mean to put the name of the hook within the brackets?
the brackets type is inconsistent